src/Serilog/Formatting/Display/LogEventLevelValue.cs

```diff
@@ -0,0 +1,99 @@
+// Copyright 2013-2015 Serilog Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+using Serilog.Events;
+using Serilog.Parsing;
+
+namespace Serilog.Formatting.Display
+{
+    // allows for the specific handling of the {Level} element.
+    // can now have a fixed width applied to it, as well as casing rules.
+    class LogEventLevelValue : LogEventPropertyValue
+    {
+        private readonly LogEventLevel _value;
+        private readonly Alignment? _alignment;
+
+        private Dictionary<LogEventLevel, string[]> _shortenedLevelMap
```

Comment:
This needs to be `static`, otherwise every rendering will end up creating one of these.

Can also be `readonly`.

May need a comment `// Assumes reading from Dictionary<,> is thread-safe after construction` (which I believe is a valid assumption currently.